### DISCUSSION TITLE
Export API - Filter based on Saved View missing

### DISCUSSION THREAD

**Discussion by Sebastian-dm** [iTwin/itwinjs-core#4946] [Q&A] [answered]
Hi

When creating an Export Connection Run through the Export API I cannot seem to find any settings for filtering based on Saved Views though the iTwin Platform API does seem to support Saved Views otherwise. This option seems to be the only option missing compared to the Export options found on iTwin Hub (see attached image).

Is this intentional or am I missing something? Is this feature just not a part of the technical preview yet? if so, do you have any idea when this will be added?

I am testing a setup of an automated batch IFC exporter, and for our project, this feature seems essential for only exporting 3D elements.

Links:
API documentation on creating an Export Run: https://developer.bentley.com/apis/export/operations/create-export-connection-run/
API documentation on getting a Saved Views: https://developer.bentley.com/apis/savedviews/operations/get-savedview/

![iTwinAPIexportSettings](https://user-images.githubusercontent.com/12986932/213675375-11dca9af-95d8-4457-bfa5-9a074fa684b9.png)

**Comment by pmconne** [comment]
@Laisvune @Aidas-Vaiksnoras

**Comment by timlawrence-bentley** [comment, marked as answer]
Hi Sebastian,

Currently the Export API does not support using saved views. We are aware of this and we do plan to add the ability to use saved views. I can't give a firm timeline on when we will complete this work, but it is unlikely in the next couple of months as we first need to complete some other updates to support this.

Tim

**Reply by Sebastian-dm** [reply]
Thank you for the reply Tim.
That is good to hear. We are really looking forward to that feature! :)
